examples/fdinteractive_fitter .py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
```

Comment:
Please delete this entire file and keep only one titled: "interactive_fitter.py"

Reply:
check out the structure here: https://github.com/rzellem/EXOTIC/tree/main/examples